op-deployer/pkg/deployer/opcm/asterisc2_test.go

```diff
@@ -10,6 +10,7 @@ import (
 )
 
 func TestNewDeployAsteriscScript(t *testing.T) {
+	t.Skip()
```

Comment:
Let's make an issue to track unskipping and reference it here in a TODO.

Reply:
[4981d13](https://github.com/ethereum-optimism/optimism/pull/15292/commits/4981d13f5fbe3f37b5118d294429ec9cdbbcb9fc)